src/mysql-util/src/decoding.rs

```diff
@@ -81,17 +85,21 @@ fn val_to_datum<'a>(
     temp_bytes: &'a mut Vec<Vec<u8>>,
     temp_jsonbs: &'a mut Vec<Jsonb>,
 ) -> Result<Datum<'a>, anyhow::Error> {
+    let column_type = match col_desc.column_type {
+        Some(ref column_type) => column_type,
+        None => anyhow::bail!("column type is not set for column: {}", col_desc.name),
```

Comment:
Ah I don't want to cause a panic - just return an error so went with this approach. It would be cool if there was shorthand for it - maybe I'm not aware of it?